### ISSUE TITLE
Correct path for existing movies when doing Libary Import

### ISSUE THREAD

**Issue by md0rca** [Radarr/Radarr#6414]
When doing a Library Import, existing movies will be completely skipped. As soon as the move already exists within Radarr, no matter if monitored or unmonitored, nor (afaict) if missing or present the import simply skips that movie.
My proposal would be to be able to change the path for the existing movie as well as unmonitor it, if wanted.

Let me explain in more detail with an example: Right now, when I have approx. 100 Stand-up shows set as wanted. I assigned them my default root folder "/comedy"and wait for them to get downloaded. I have to add that I am using genre subfolders and circumvent the lack of support for these by adding each folder as a root folder in Radarr. 

In parallel I record shows on TV and some 50 of these shows land on another hard drive. Then I decide to join the new shows with everything else on the main server. The only (imho) non-manual way to do that is to copy the new shows here they belong and then import them via Library Import. But some of them will be ignored, since Radarr is already looking for them.

In the Library Import dialogue these files resp. the shows are recognized immediately, so it is not an issue of Radarr not knowing what do do with them or what shows they are.

The same problem occurs, if I get a well ordered and named library from a friend. Instead of destroying the meticulously sorted movie library by pouring all files into SABnzbd's download folder, I would rather keep the structure and import genre by genre (i.e. folder by folder) into Radarr. I would have expected that Radarr allows me to import them and that it simply changes the original root folder and path to the new path. Or that it at least allows my to import those files and unmonitor the respective shows/movies. But I have found no way, how to accomplish this and thus this feature request.

So in other, very simple words, Radarr is looking for these shows, I tell Radarr where they are and Radarr ignores them ;-)

Did I explain this well enough? 

![screenshot](https://user-images.githubusercontent.com/4022167/123147685-9f07c400-d45f-11eb-857d-571e619cfa42.png)

**Comment by bakerboy448** [comment]
Movie Index -> Manual Import sounds like what you want?

I.e. parse through this folder and map up the movie files and then import into the library

Really the only time you'd need to remap a movie is if you assigned a folder to the wrong movie.

**Comment by fryfrog** [comment]
While you wait for someone to care enough to develop this feature... here is how I would accomplish what you're after, I think.

Use a merge file system so that you don't have to worry about files/folders on different drives. Aim a Manual Import at the folder in question and use *move* to import anything that is an upgrade, this will take care of items *already* in your library. Then you can move them to a library folder and Library Import on it to pick up *new* things. That does still leave duplicates of the *old* things that *weren't* an upgrade though. I don't have a brilliant way to solve that.

**Comment by fryfrog** [comment]
And for the disappointing part, I suspect that you could probably learn .NET and work w/ the developers to improve Radarr in some ways that would help this use case before they get around to it themselves. :(

**Comment by md0rca** [comment]
The manual import will always move the files to the assigned root folder, so it would destroy the folder structure and rename the folders themselves. I understand that the folder and naming scheme is based on Sonarr and followed many of the discussions about that issue. I also understand that this resp. all that is being asked is too much for a handful developers. But since I cannot contribute in a more productive way I can only ask or suggest.
To be honest, I pondered adding this issue as a bug report not a feature request, since I think pointing Radarr to bunch of movies and it ignores those it is looking for is a real bug - but I was afraid someone was gonna bite my head of :)

**Comment by fryfrog** [comment]
I doubt you'll ever get exactly what *you* want here because it is very clearly super niche. But I think parts of your ideas are interesting and I wouldn't be surprised if *some* of it ends up somewhere *eventually*. But what you describe sounds like a *lot* of work and would basically only be for you. And a bunch of what you want can already be done w/ different bits and pieces, just not quite how you like.

**Comment by md0rca** [comment]
I am not so sure about "just me". Look at it this way: Someone has a huge, well-groomed library and joins the Radarr crowd. Coming from CP he also has quite a few movies he's still looking for. Now he can only throw all his movies into one folder OR he could import the library (root)folder by (root)folder, if Radarr doesn't skip all those files it already knows. At the very least it could offer to unmonitor the existing movies. This way Radarr wouldn't try to download files already on the hard drive.
I get that changing the path might be more difficult, but removing the monitored flag for recognized movie during library import sound much easier, right?

**Comment by md0rca** [comment]
I just thought of another way (and this is what I will be doing manually now): During library import Radarr could offer the possibilty to remove those existing movies. This way it can import the files on the next import run. Not sure about the repercussions though, but it works for me.
